website/docs/getting-started/pulling-an-image.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 2
+title: Pulling an image
+description: Pulling an image to your container engine.
+keywords: [podman desktop, podman, containers, images]
+tags: [images]
+---
+
+# Pulling an image to your Container engine
+
+With Podman Desktop, you can pull an image from a registry, to your container engine.
+
+#### Prerequisites
+
+- The image is available in a registry.
+- If the registry or the image are not publicly available, you configured access to the registry on Podman Desktop in **<icon icon="fa-solid fa-cog" size="lg" /> Settings > registry**.
```

Comment:
```suggestion
- If the registry or the image are not publicly available, you need to configure access to the registry on Podman Desktop in **<icon icon="fa-solid fa-cog" size="lg" /> Settings > registry**.
```

Reply:
No, it's a prerequisite. Past form is correct.